src/photos/ducks/timeline/index.jsx

```diff
@@ -19,12 +19,12 @@ const TIMELINE_QUERY = client =>
       'metadata.datetime': 'desc'
     })
 
-const TIMELINE_MUTATIONS = query => ({
+const TIMELINE_MUTATIONS = (client, query) => ({
   uploadPhoto: async (file, dirId) => {
-    return query.client.mutate({
+    return client.mutate({
       mutationType: 'UPLOAD_PHOTO',
       execute: () =>
-        query.client.collection('io.cozy.files').createFile(file, { dirId })
+        client.collection('io.cozy.files').createFile(file, { dirId })
     })
   },
   deletePhoto: photo => query.client.destroy(photo)
```

Comment:
deletePhoto methods will still work? Or do we need to use `deletePhoto: photo => client.destroy(photo)`

Reply:
Still worked, but I'm changing it anyway to stay consistent.